packages/fxa-shared/metrics/amplitude.js

```diff
@@ -442,3 +443,18 @@ function mapLocation(location) {
     };
   }
 }
+
+function estimateTime(times) {
+  const skew = times.received - times.sent;
+  return times.start + times.offset + skew;
+}
```

Comment:
https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/server/lib/flow-event.js#L70 can make use of this now.